### DISCUSSION TITLE
Manually saving to flash or save before an OTA update

### DISCUSSION THREAD

**Discussion by tommyhtran** [emporia-vue-local/esphome#323] [General]
I'd like to be able to save my daily energy sensor values before any planned shutdown or restart, such as when doing an OTA update.

```flash_write_interval``` doesn't achieve what I'm looking for and the config template suggests setting `restore: false`, in contrast to the recommendation (https://github.com/esphome/feature-requests/issues/461#issuecomment-1387127619) to setting it to ```true```.

Are there any options to achieve this?

**Comment by t0ny-peng** [comment]
I don't think it's a good idea to use VUE2or3 as data persistence as it's not as reliable as home assistant or other home automation platform. Better to treat VUE2 as an ephemeral data provider and have the storage logic somewhere.
